plugins/zoho_crm.md

```diff
@@ -0,0 +1,21 @@
+# Mautic - Zoho CRM plugin
+
+This plugin can push a lead to Zoho CRM when a lead makes some action.
+
+If you don't have the Zoho CRM account yet, [create it](https://www.zoho.com/crm/).
+
+## Configure the Zoho CRM plugin
+
+Insert the email and password you created the Zoho account with into the Mautic Zoho integration plugin and authorize it. Set the *Publish* switch to *Yes*. Save.
```

Comment:
Might mention here that if Zoho Two Factor Authentication is enabled, an Application Specific Password will need to be generated and used. https://www.zoho.com/mail/help/adminconsole/two-factor-authentication.html#alink5